myna/application/rve/rve_selection/postprocess.py

```diff
@@ -58,8 +59,8 @@ def main(argv=None):
             part = f"{row.part}"
             region = f"rve_{row.id}"
             settings["data"]["build"]["parts"][part]["regions"][region] = {
-                "X (m)": row._1,
-                "Y (m)": row._2,
+                "x": row._1,
```

Comment:
The "x" and "y" labels are what are expected for the region location currently (no units). This was a typo/bug. 

Changing it to include units would break the input file, unless I add an option to allow units when loading the dictionary with some parsing in `myna.core.workflow.load_input.load_input()`. Could do that in a separate PR.